runner/src/server/services/uploadService.ts

```diff
@@ -224,4 +242,31 @@ export class UploadService {
     const promises = paths.map((path) => get<string>(path, {}));
     return Promise.all(promises);
   }
+
+  async uploadFilesS3(files) {
+    let response = new Array();
+
+    for (const file of files) {
+      const uploadParams = {
+        Bucket: bucketName,
+        Body: file,
+        Key: file.hapi.filename,
```

Comment:
Can we prefix this key with the `application_id`? e.g. 
```application_id + '/' + `file.hapi.filename``` ?
This will help us identify it later on.

Reply:
prefixing key with application name